docs/antora.yml

```diff
@@ -19,7 +19,7 @@ asciidoc:
 
     # External URLs
     link-quarkus-code-generator: code.quarkus.io
-    quarkus-cxf-project-name: CXF Extensions for Quarkus
+    quarkus-cxf-project-name: CXF
```

Comment:
This should stay as is because it is the full project name that we use throughout the docs.